openlibrary/templates/home/scroll-to-top.html

```diff
@@ -0,0 +1,44 @@
+$def with (test)
+<style>
+#myBtn {
```

Comment:
Embedded CSS is strongly discouraged. @jdlrobson may be able to suggest a good home for it.

Reply:
I have removed the pre-existing return to top button in my last commit